lib/diagnostic.rb

```diff
@@ -13,28 +13,31 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.each do |prev_item, nex_item|
```

Comment:
`.each` doesn't give blocks two arguments, only one - the element that we're currently looking at.